docs/ssh/main.md

```diff
@@ -111,15 +114,19 @@ Example 1-b: Copying a single file from Jane's personal computer (client) to her
 
 Example 2-a: Copying a directory (collection of files) from Jane's personal computer (client) to Dr. John's `project` directory:
 
-`scp tests -r jdoe@midway3.rcc.uchicago.edu:/project/drpepper/users/jdoe/`
+`scp -r tests jdoe@midway3.rcc.uchicago.edu:/project/drpepper/users/jdoe/`
+
+On MacOS, you need to add ```-O``` if there is no folder with the same name on the target server:
+
+`scp -O -r tests jdoe@midway3.rcc.uchicago.edu:/project/drpepper/users/jdoe/`
```

Comment:
tested Windows; can't test on Mac as i don't have access to one

Reply:
I tested this on my MacOS.